### ISSUE TITLE
first-line-h1 insufficient replacement for first-heading-h1 

### ISSUE THREAD

**Issue by bbodenmiller** [DavidAnson/markdownlint#1416]
There is times that the first line is not desired as a header (e.g. a TOC) but that we want to ensure the first header (which follows TOC) is in fact H1. This does not appear possible with `first-line-h1`

**Comment by DavidAnson** [comment]
Agreed. I'm not sure the behavior you describe is entirely in keeping with the intent of MD041/first-line-heading, anyway.

However, it could be a reasonable thing for MD001/heading-increment to enforce, assuming it got a new parameter starting_level which would be set to 1 in your case.

https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md

**Comment by bbodenmiller** [comment]
@DavidAnson starting_level option on MD001/heading-increment would certainly work!
